src/trading/suggestSlippageBpsFromFee.ts

```diff
@@ -0,0 +1,94 @@
+import { applyPercentage, percentageToBps } from 'src/common/utils/math'
+
+const MIN_SLIPPAGE_BPS = 50
```

Comment:
If we don't have a cap of the max slippage (other than 100%), should we floor on the min slippage suggested?
For example, in case it's a large stable trade, does it still make sense to suggest 0.5%?